src/poparser.js

```diff
@@ -4,18 +4,29 @@ import { Transform } from 'readable-stream';
 import util from 'util';
 
 /**
- * Po parser options
+ * @typedef {import('stream').Stream.Writable} WritableState
+ * @typedef {import('./types.js').GetTextTranslations} GetTextTranslations
+ * @typedef {import('./types.js').GetTextTranslation} GetTextTranslation
+ * @typedef {import('./types.js').GetTextComment} GetTextComment
+ * @typedef {import('./types.js').Translations} Translations
+ */
+
+/**
  * @typedef {{ defaultCharset?: string, validation?: boolean }} Options Po parser options
- *
- * The single Node object in the PO file
- * @typedef {{
- * key?: string,
- * type?: number,
- * value: string,
- * quote?: string,
- * obsolete?: boolean,
- * comments?: import('./types.js').GetTextComment | undefined
- * }} Node PO node
+ */
+
+/**
+ * @typedef {(...args: any[]) => void} DoneCallback
+ */
+
+/**
+ * @typedef {Object} Node A single Node object in the PO file
+ * @property {string} [key]
+ * @property {number} [type]
+ * @property {string} value
+ * @property {string} [quote]
+ * @property {boolean} [obsolete]
+ * @property {GetTextComment | undefined} [comments]
```

Comment:
All the other interface types are declared this way, so I changes this one to match.